app/src/test/java/com/github/multimatum_team/multimatum/AddDeadlineTest.kt

```diff
@@ -91,16 +102,38 @@ class AddDeadlineTest {
         onView(withId(R.id.add_deadline_select_time))
             .perform(ViewActions.click())
         val timeDialog = ShadowTimePickerDialog.getLatestDialog() as TimePickerDialog
-        timeDialog.updateTime(10,10)
+        timeDialog.updateTime(10, 10)
         timeDialog.getButton(TimePickerDialog.BUTTON_POSITIVE).performClick()
 
         // Check if time is correctly selected
         assertEquals("10:10", getText(withId(R.id.add_deadline_text_time)))
 
+        onView(withId(R.id.radio_notification_1h)).perform()
+
         // Check if Toast correctly appear
         onView(withId(R.id.add_deadline_button)).perform(ViewActions.click())
-        MatcherAssert.assertThat(ShadowToast.getTextOfLatestToast(),
-            CoreMatchers.equalTo(RuntimeEnvironment.getApplication().applicationContext.getString(R.string.deadline_created)))
+        MatcherAssert.assertThat(
+            ShadowToast.getTextOfLatestToast(),
+            CoreMatchers.equalTo(RuntimeEnvironment.getApplication().applicationContext.getString(R.string.deadline_created))
```

Comment:
You can get the `applicationContext` once and at the top of the class and simply use a variable for it, it would be more readable